ledger/eval/eval.go

```diff
@@ -788,6 +791,39 @@ func StartEvaluator(l LedgerForEvaluator, hdr bookkeeping.BlockHeader, evalOpts
 		return nil, fmt.Errorf("overflowed subtracting rewards for block %v", hdr.Round)
 	}
 
+	// Move last block's proposer payout to the proposer
+	if !prevHeader.Proposer.IsZero() {
+		// Payout the previous proposer.
+		if !prevHeader.ProposerPayout.IsZero() {
```

Comment:
Sanity check: why do we care if the previor blocks proposer payout was zero? (Obvious why the proposer needs to be non-zero).